index.html

```diff
@@ -60,7 +60,7 @@
 			Your slides don't have to be this way, and it doesn't take a whole lot of effort to make them better.
 		</p>
 		<p>
-			PowerPointLabs makes creating engaging PowerPoint presentations easy. <strong>Check out what it can do for you below.</strong>
+			PowerPointLabs is an <a href="https://github.com/PowerPointLabs/PowerPointLabs"><strong>open source project</strong></a> based in the <a href="http://www.comp.nus.edu.sg/">National University of Singapore</a>. It makes creating engaging PowerPoint presentations easy. <strong>Check out what it can do for you below.</strong>
```

Comment:
It would be better if we can link to HTTPS versions of the websites if possible.